sampleapplication/src/main/java/com/evervault/sampleapplication/sample/PaymentCardSamples.kt

```diff
@@ -0,0 +1,175 @@
+package com.evervault.sampleapplication.sample
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldColors
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.evervault.sdk.input.model.label.LabelTextsDefaults
+import com.evervault.sdk.input.model.placeholder.PlaceholderTextsDefaults
+import com.evervault.sdk.input.ui.PaymentCardInputScope
+
+/**
+ * This file is a partial copy of the PaymentCardWithCustomizedLayoutSample and private methods
+ * from com.evervault.sdk.input.ui.sample.PaymentCardSamples
+ *
+ * In the future we should create a shared module so some of the components used for previews can be
+ * shared without exposing them to the clients with public visibilities
+ */
+
+@Composable
+internal fun PaymentCardInputScope.PaymentCardCustomLayoutWithNewComponents(modifier: Modifier) {
```

Comment:
Needed to create this one not to pollute the `NavigationGraph`. Temporarily until we create a shared module where we can share components between the libs and the sample app